docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -12,6 +12,7 @@
 - [Accessing BigCommerce Data](#accessing-bigcommerce-data)

```

Comment:
These links should be sentence case and the headers should match the sentence case.

Reply:
Ok, changed existing TOC and headers to sentence case